tests/rules/javascript/test.no_eval.js

```diff
@@ -0,0 +1,114 @@
+import { VALIDATION_WARNING } from 'const';
+import JavaScriptScanner from 'scanners/javascript';
+
+// These rules were mostly copied and adapted from
+// https://github.com/eslint/eslint/blob/master/tests/lib/rules/no-eval.js
+// Please make sure to keep them up-to-date and report upstream errors.
+describe('no_eval', () => {
+  var validCodes = [
+    'Eval(foo)',
+
+    // User-defined eval methods.
+    'window.noeval("foo")',
+    'function foo() { var eval = "foo"; window[eval]("foo") }',
+    'global.eval("foo")',
+    'global.noeval("foo")',
+    'function foo() { var eval = "foo"; global[eval]("foo") }',
```

Comment:
hehe, it actually took me a few minutes to spot this after sublime was sure it's not identical 🤣